meta-flutter/README.md

```diff
@@ -39,5 +55,33 @@ You need to install libsystemd in the same way with the DRM-GBM backend.
 $ bitbake flutter-drm-eglstream-backend
 ```
 
+## SDK toolchain
+If you use the SDK, please follow the steps below.  
+Putting the following in your conf/local.conf :
+```
+CLANGSDK = "1"
+```
+See also: [Adding clang in generated SDK toolchain](https://github.com/kraj/meta-clang/blob/master/README.md#adding-clang-in-generated-sdk-toolchain)
+
+The SDK on yocto is built with the following command : 
+```Shell
+$ bitbake core-image-weston -c populate_sdk
+```
+See also: [SDK building an sdk installer](https://www.yoctoproject.org/docs/2.1/sdk-manual/sdk-manual.html#sdk-building-an-sdk-installer)
+
+Install the SDK :  
+e.g. aarch64
+```Shell
+$ ./tmp/deploy/sdk/poky-glibc-x86_64-core-image-weston-aarch64-qemuarm64-toolchain-3.1.7.sh
+```
+
+This `cross-toolchain.cmake` is a sample of aarch64. Please set the variable in your environment to match the location of your SDK installation :  
```

Comment:
This `cross-toolchain.cmake` -> `[cross-toolchain.cmake](URI)`

Reply:
`Please set the variable in your environment to match the location of your SDK installation`

What does it mean?